ntbs-service/Services/ReportingLinksService.cs

```diff
@@ -23,8 +23,15 @@ public ReportingLinksService(IConfiguration configuration)
 
         public string GetClusterReport(string clusterId)
         {
-            // TODO Implement this properly in NTBS-2249
-            return "";
+            var clusterReportBase = _externalLinks.ClusterReportBase;
+            var clusterReportReplacementSymbol = _externalLinks.ClusterReportReplacementSymbol;
+
+            if (string.IsNullOrEmpty(clusterReportBase) || string.IsNullOrEmpty(clusterReportReplacementSymbol))
+            {
+                return "";
```

Comment:
Semantically, I think it would be slightly better to return `null` here

Reply:
Yep I agree